### PR TITLE
Add missing endpoints

### PR DESCRIPTION
**Pull request by Slawisch**

### CODE REVIEW

**Review by proskurnicha — APPROVED** [review]

**Comment by Slawisch** [comment]
I'll resolve conflicts in the morning🙂

File: backend/ApiGateway/ApiGateway/configuration.Production.json [hunk 1]
```diff
@@ -71,6 +71,62 @@
       "UpstreamPathTemplate": "/api/Songs/search?searchTerm={searchTerm}",
       "UpstreamHttpMethod": [ "GET" ]
     },
+    {
+      "DownstreamPathTemplate": "/api/Songs/liked",
+      "DownstreamScheme": "https",
+      "DownstreamHostandPorts": [
+        {
+          "Host": "localhost",
+          "Port": "5001"
```
Comment (NikitaKrasnovSW) [inline backend/ApiGateway/ApiGateway/configuration.Production.json]: Production config should not contain localhost. Please pay attention on other routes in this file

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]

File: backend/ApiGateway/ApiGateway/configuration.Production.json (hunk 1 above)
Comment (Slawisch) [reply backend/ApiGateway/ApiGateway/configuration.Production.json]: Oh, sure, thanks

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]
Need http instead of https

**Review by NikitaKrasnovSW — APPROVED** [review]

**Pull request merged**